src/AppRouter.tsx

```diff
@@ -45,7 +46,7 @@ export const AppRouter = () => {
           element={
             <>
               <SearchBar />
-              <CategoryDetailsComponent />
+              <RecipeList />
```

Comment:
This does not seem to be a right place for CategoryDetailsComponent, replaced with recipe list.